### PR TITLE
Explain acceptable bundles

### PR DESCRIPTION
**Pull request by lcarva**
https://issues.redhat.com/browse/HACBS-1236

Signed-off-by: Luiz Carvalho <lucarval@redhat.com>

### CODE REVIEW

**Comment by lcarva** [comment]
I need help verifying these changes are rendered properly. Not sure how to do it.

**Comment by lcarva** [comment]
@cuipinghuo, please let me know what you think!

File: policy/pipeline/task_bundle.rego [hunk 1]
```diff
@@ -79,7 +79,8 @@ warn[result] {
 #   are acceptable given the tracked effective_on date. See the list of acceptable
 #   task bundles at xref:acceptable_bundles.adoc#_task_bundles[Acceptable Bundles] or look at
 #   link:https://github.com/hacbs-contract/ec-policies/blob/main/data/acceptable_tekton_bundles.yml[data/acceptable_tekton_bundles.yml]
-#   in this git repository.
+#   in this git repository. The meaning of an acceptable bundle is explained in
+#   xref:acceptable_bundles.adoc#_task_bundles[Acceptable Bundles]
```
Comment (cuipinghuo) [inline policy/pipeline/task_bundle.rego:83]: @lcarva Does is it also necessary to update the meaning of acceptable bundle for line 65 of ` Task bundle is out of date`?

File: policy/release/attestation_task_bundle.rego [hunk 2]
```diff
@@ -63,7 +63,8 @@ warn[result] {
 #   is the most recent acceptable one. See the list of acceptable
 #   task bundles at xref:acceptable_bundles.adoc#_task_bundles[Acceptable Bundles] or look at
 #   link:https://github.com/hacbs-contract/ec-policies/blob/main/data/acceptable_tekton_bundles.yml[data/acceptable_tekton_bundles.yml]
-#   in this git repository.
+#   in this git repository. The meaning of an acceptable bundle is explained in
+#   xref:acceptable_bundles.adoc#_task_bundles[Acceptable Bundles]
```
Comment (cuipinghuo) [inline policy/release/attestation_task_bundle.rego:67]: @lcarva Do you mean to update the meaning of acceptable bundle for line 84 of task title `Task bundle is not acceptable`?

**Comment by simonbaird** [comment]
It's quite complicated, but I think well explained.

Not sure the answer to @cuipinghuo 's question, but let's assume it could be addressed in a later PR.

Merge?

**Review by simonbaird — APPROVED** [review]

File: policy/pipeline/task_bundle.rego (hunk 1 above)
Comment (lcarva) [reply policy/pipeline/task_bundle.rego:83]: https://github.com/hacbs-contract/ec-policies/pull/167

File: policy/release/attestation_task_bundle.rego (hunk 2 above)
Comment (lcarva) [reply policy/release/attestation_task_bundle.rego:67]: https://github.com/hacbs-contract/ec-policies/pull/167

**Pull request merged**